starcheck/src/starcheck

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-unset LD_LIBRARY_PATH
-unset SYBASE
+# Don't unset LD_LIBRARY_PATH and SYBASE because 'perl' below
```

Comment:
I don't think we need a 'perl3ska' or the like.  If this is always being called from another wrapper we control, we can call flt_envs and set SYBASE and LD_LIBRARY_PATH appropriately just before this.  If there is anything from the MP environment, it should just end up further down the path.